webroot/src/network/userApi/data.api.ts

```diff
@@ -0,0 +1,177 @@
+//
+//  user.api.ts
+//  CompactConnect
+//
+//  Created by InspiringApps on 6/18/24.
+//
+
+import axios, { AxiosInstance } from 'axios';
+import {
+    requestError,
+    requestSuccess,
+    responseSuccess,
+    responseError
+} from '@network/userApi/interceptors';
+import { config as envConfig } from '@plugins/EnvConfig/envConfig.plugin';
+import { UserSerializer } from '@models/User/User.model';
+
+export interface RequestParamsInterfaceLocal {
+    compact?: string;
+    search?: string;
+    pageSize?: number;
+    pageNumber?: number;
+    lastKey?: string;
+    prevLastKey?: string;
+    sortBy?: string;
+    sortDirection?: string;
+}
+
+export interface DataApiInterface {
+    api: AxiosInstance;
+}
+
+export class UserDataApi implements DataApiInterface {
+    api: AxiosInstance;
+
+    public constructor() {
+        // Initial Axios config
+        this.api = axios.create({
+            baseURL: envConfig.apiUrlUser,
+            // withCredentials: true,
```

Comment:
can we nix this?

Reply:
Yep. Wasn't sure until more recently whether cookies would be in play. Removed.